python/mxnet/image/image.py

```diff
@@ -178,6 +233,59 @@ def scale_down(src_size, size):
     return int(w), int(h)
 
 
+def copyMakeBorder(src, top, bot, left, right, *args, **kwargs):
```

Comment:
Can you rename ```bot``` to ```bottom``` ?

Reply:
No, that would break the API, as said in the PR description

> Though mx.image.imresize and mx.image.copyMakeBorder is currently available for external use, it does not show up in the docs because it was hidden behind an internal function call.